scheduled_sweeper.bat

```diff
@@ -0,0 +1,2 @@
+call activate sweeper
+sweeper sweep --workspace=\\itwfpcap2\AGRC\sgid_to_agol\ConnectionFilesSGID\SGID_internal\SGID_agrc.sde --change-detect --try-fix --scheduled --save-report=C:\Temp
```

Comment:
**nitpick**: I don't think that we want the share name in this public repo. It should probably be moved to `credentials.py` or something.

Reply:
What do you think about using the forklift garage instead of the share?

Reply:
Yes, I believe we are actually using the connection files in the forklift garage when running from the server.  In talking with Jake, it sounds like the .bat is usually specific to the machine it's run on (in this case, mine).  The .bat on the server will point to the connection file that sees all of SGID, but the code then points to the connection files with edit permission using a dictionary in `credentials.py` as a lookup.  Should I just add the .bat file to the gitignore so it's not tracked in the repo?

Reply:
yes i think that it is a good idea to ignore this. you'll have to delete it from git tracking as well.

Reply:
Done